modules/aws/ecr.go

```diff
@@ -0,0 +1,97 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecr"
+	"github.com/gruntwork-io/gruntwork-cli/errors"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
+// CreateECRRepo creates a new ECR Repository. This will fail the test and stop execution if there is an error.
+func CreateECRRepo(t testing.TestingT, region string, name string) *ecr.Repository {
+	repo, err := CreateECRRepoE(t, region, name)
+	require.NoError(t, err)
+	return repo
+}
+
+// CreateECRRepoE creates a new ECR Repository.
+func CreateECRRepoE(t testing.TestingT, region string, name string) (*ecr.Repository, error) {
+	client := NewECRClient(t, region)
+	resp, err := client.CreateRepository(&ecr.CreateRepositoryInput{RepositoryName: aws.String(name)})
+	if err != nil {
+		return nil, err
+	}
+	return resp.Repository, nil
+}
+
+// GetECRRepo gets an ECR repository by name. This will fail the test and stop execution if there is an error.
+func GetECRRepo(t testing.TestingT, region string, name string) *ecr.Repository {
+	repo, err := GetECRRepoE(t, region, name)
+	require.NoError(t, err)
+	return repo
+}
+
+// GetECRRepoE gets an ECR Repository by name.
```

Comment:
NIT: It's probably worth mentioning that this will error if the ECS repo doesn't exist.

Reply:
2e7a21e